configure.ac

```diff
@@ -743,6 +743,9 @@ if test ! -L "$srcdir"/include/alsa ; then
   ln -sf . "$srcdir"/include/alsa
 fi
 
+dnl enable large file support
+CFLAGS="${CFLAGS} -D_FILE_OFFSET_BITS=64"
```

Comment:
Does `AC_SYS_LARGEFILE` work instead?

Reply:
I tried that first as it seemed promising, and it didn't address the issue. As I recall that flag is already enabled in alsa-utils for example, but I required additional changes there: https://github.com/alsa-project/alsa-utils/pull/223

It's possible it could work with some additional configuration changes I was missing though. Let me know if you have any suggestions.